model/dns_zone/dns_zone.rb

```diff
@@ -5,6 +5,7 @@
 
 class DnsZone < Sequel::Model
```

Comment:
There are several aspects in the commit message that confuse me, e.g. why we're mucking so much with systemd on the server hosting the DNS server (perhaps only bind the external interface). I also think the selection criteria lacks the key theory that cinched it for knot: incremental updates, instead of batch processing/preprocessing of zone definition.

Reply:
> perhaps only bind the external interface, I take it the problem is conflict with loopback binding port 53
Not sure how to do that, knot only takes list of ip addresses to listen to. Though the documentation lists this as a feature "Binding to non-local addresses (IP_FREEBIND on Linux, IP_BINDANY/IPV6_BINDANY on FreeBSD)". So, I'll look into that.

> I also think the selection criteria documentation in the commit message lacks the key theory that cinched it for knot (pun not intended, but not...not intended): incremental updates, instead of batch processing/preprocessing of zone definition.

I'll add that to the commit message.

Reply:
Updated the commit message, both for interface and for reason to pick knot.